src/dmd/backend/barray.d

```diff
@@ -31,27 +31,29 @@ struct Barray(T)
      */
     void setLength(size_t length)
     {
-        if (capacity < length)
+        static void enlarge(ref Barray barray, size_t length)
         {
-            if (capacity == 0)
-                capacity = length;
-            else
-                capacity = length + (length >> 1);
-            capacity = (capacity + 15) & ~15;
-            T* p = cast(T*)realloc(array.ptr, capacity * T.sizeof);
+            pragma(inline, false);
```

Comment:
I thought this goes on the function declaration, not in the function body?

Reply:
Either works.